src/components/common/SliderGridColumn/SliderGridColumn.css

```diff
@@ -62,7 +62,7 @@
     cursor: pointer;
 }
 
-@media screen and (min-width: 1105px) {
+@media screen and (min-width: 450px) {
```

Comment:
Why this changed?

Reply:
To not be able to hover it only on the phones, because laptop also can have a small screen that's why it's not 1105px